.gitignore

```diff
@@ -14,6 +14,7 @@ coverage
 test-results.xml
 backend-coverage
 backend/cfy-config-softlayer-old
+backend/cfy-config-softlayer
```

Comment:
?Please have her comment the answer.

Thanks,

Guy.

________________________________
From: SefiGiga <notifications@github.com>
Sent: Wednesday, February 4, 2015 11:08 AM
To: CloudifySource/cloudify-widget-ui
Cc: Guy Mograbi
Subject: Re: [cloudify-widget-ui] Cw 298 widget execution refactoring (#89)


In .gitignore<https://github.com/CloudifySource/cloudify-widget-ui/pull/89#discussion_r24070384>:

> @@ -14,6 +14,7 @@ coverage
>  test-results.xml
>  backend-coverage
>  backend/cfy-config-softlayer-old
> +backend/cfy-config-softlayer


ask liron, I just followed her instructions.

-
Reply to this email directly or view it on GitHub<https://github.com/CloudifySource/cloudify-widget-ui/pull/89/files#r24070384>.